bin/brew

```diff
@@ -23,7 +23,7 @@ unset GEM_HOME
 unset GEM_PATH
 
 BREW_SYSTEM=$(uname -s | tr "[:upper:]" "[:lower:]")
-if [ "$BREW_SYSTEM" = "darwin" ]
+if [ "$BREW_SYSTEM" = "darwin" ] && [ "$HOMEBREW_OLD_RUBY_TEST" != "1.8.7" ]
```

Comment:
Maybe just whether the env exists rather than its content. After all, its content is never used.